### DISCUSSION TITLE
Unable to install Neovim HEAD with brew, missing LUV_INCLUDE_DIR error

### DISCUSSION THREAD

**Discussion by yriveiro** [neovim/neovim#24290] [Q&A]
Hi,

Does anyone get this error trying to install Neovim from `HEAD` on Homebrew? Any clue of how to fix it?

```
==> cmake -S . -B build -DLUV_LIBRARY=/opt/homebrew/opt/luv/lib/libluv.dylib -DLIBUV_LIBRARY=/opt/homebrew/opt/libuv/lib/libuv.dylib
Last 15 lines from /Users/yriveiro/Library/Logs/Homebrew/neovim/01.cmake:
-- Looking for sendfile in sendfile - not found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found Libuv: /opt/homebrew/opt/libuv/lib/libuv.dylib (Required is at least version "1.28.0")
CMake Error at /opt/homebrew/Cellar/cmake/3.26.4/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Luv (missing: LUV_INCLUDE_DIR) (Required is at least version
  "1.43.0")
Call Stack (most recent call first):
  /opt/homebrew/Cellar/cmake/3.26.4/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindLuv.cmake:12 (find_package_handle_standard_args)
  src/nvim/CMakeLists.txt:24 (find_package)
```

**Comment by ziod** [comment]
Hi, @yriveiro 
It seems like you don't have sufficient packages. Please install them.
Here is the example for Debian Linux:
`sudo apt install lua-luv-dev lua-lpeg libtree-sitter-dev libunibilium-dev libutf8proc-dev`

**Reply by alerque** [reply]
This misses the mark: `brew` is a package manager and takes care of all dependencies, including for HEAD builds.

If (and this is a big if) there is an issue it should be reported to Homebrew where it can be fixed, but no the OP does not need to install more packages on their own.

**Comment by clason** [comment]
The real answer is: distro `HEAD` builds are not supported; neither by us nor by homebrew. Caveat emptor.

(Closing as stale.)
